src/timeAgo.ts

```diff
@@ -1,16 +1,5 @@
 type Unit = 's' | 'm' | 'h' | 'd';
 
-const dayOfWeek = (day: number): string =>
-	[
-		'Sunday',
-		'Monday',
-		'Tuesday',
-		'Wednesday',
-		'Thursday',
-		'Friday',
-		'Saturday',
-	][day];
-
 const shortMonth = (month: number): string =>
```

Comment:
Even though this happens to be exactly the first 3 characters of `longMonth`, I think this makes reading easier than a substring. <3